moduleframework/helpers/container_helper.py

```diff
@@ -37,41 +37,50 @@ def __init__(self):
         set basic object variables
         """
         super(ContainerHelper, self).__init__()
-        static_name="testcontainer"
+        static_name = "testcontainer"
         self.tarbased = None
-        self.jmeno = None
+        self.name = None
```

Comment:
+1

Reply:
@TomasTomecek @phracek I'm not against. but I would like to change it to something different, like
`self.some_own_identifier`. I've choosed `self.jmeno` before because it is not name and should not lead to `name` interpretation, it is something what is close to `name` but this is not name, but more less some more less unique identifier for various artifacts.